.github/workflows/ci-build.yaml

```diff
@@ -102,6 +102,7 @@ jobs:
               - .mlc_config.json
               - .spelling
               - mkdocs.yml
+              - hack/patch-nix-hash.sh
```

Comment:
this list is categorized by comments -- this is currently placed under the docs scripts, which wouldn't be correct.

I'd move it up to after `.golangci.yml` and add its own comment